data/ecosystems/x/xrp.toml

```diff
@@ -1679,3 +1679,1701 @@ url = "https://github.com/zshahzadpumacy/Redimi-Giftcards-SmartContract"
+
+[[repo]]
+url = "https://github.com/XinFinOrg/GravitasICO"
+
+[[repo]]
+url = "https://github.com/justmoon/ripple-push-notifier"
+
+[[repo]]
+url = "https://github.com/you21979/node-ripple-lib-promise"
+
+[[repo]]
+url = "https://github.com/ajkagy/xls20-bridge-webapp"
+
+[[repo]]
+url = "https://github.com/xdv-testing/testnet-faucet"
+
+[[repo]]
+url = "https://github.com/onxrp/onXRP-nft-twitter-bot"
+
+[[repo]]
+url = "https://github.com/instagold/republicoftexas.money"
+
+[[repo]]
+url = "https://github.com/xdv/divvyd-historical-database"
+
+[[repo]]
+url = "https://github.com/StevenRB7/dparcheweb"
+
+[[repo]]
+url = "https://github.com/Transia-RnD/xrpl-helpers"
+
+[[repo]]
+url = "https://github.com/rockwalletcode/xrpl4j"
+
+[[repo]]
+url = "https://github.com/wqmer/Ripple_RPC_Middleware"
+
+[[repo]]
+url = "https://github.com/xdvsf/xrpl-dev-portal"
+
+[[repo]]
+url = "https://github.com/crossmarkio/crosskit"
+
+[[repo]]
+url = "https://github.com/Coin-Tunnel/Coin-Tunnel"
+
+[[repo]]
+url = "https://github.com/sublimator/ripple-core"
+
+[[repo]]
+url = "https://github.com/ygolde/Eos-based-exchange"
+
+[[repo]]
+url = "https://github.com/InternetExplorer7/rwallet-server"
+
+[[repo]]
+url = "https://github.com/owenkellogg/ripple-altnet-payment-robot"
+
+[[repo]]
+url = "https://github.com/keepkey/keepkey-sdk"
+
+[[repo]]
+url = "https://github.com/dino-rodriguez/easy-connector-bundle"
+
+[[repo]]
+url = "https://github.com/develoQ/xrpl-pathfinding-visualization"
+
+[[repo]]
+url = "https://github.com/iartem/blockchain-integration"
+
+[[repo]]
+url = "https://github.com/PALAXO/ledgerconnector"
+
+[[repo]]
+url = "https://github.com/ultiledger/hd-wallet"
+
+[[repo]]
+url = "https://github.com/XRPL-Labs/hookscript"
+
+[[repo]]
+url = "https://github.com/orbit-chain/bridge-dockerize"
+
+[[repo]]
+url = "https://github.com/BitGo/key-recovery-service-v2"
+
+[[repo]]
+url = "https://github.com/ripplefox/ripplewallet"
+
+[[repo]]
+url = "https://github.com/mlaroy/xrp-balance"
+
+[[repo]]
+url = "https://github.com/seetadev/EtherGlance"
+
+[[repo]]
+url = "https://github.com/ToastWallet/core"
+
+[[repo]]
+url = "https://github.com/xdv-testing/ripple-domain-verifier"
+
+[[repo]]
+url = "https://github.com/MarcSagues/frontend-technical-challenge"
+
+[[repo]]
+url = "https://github.com/rus7hex/secux-devdocs"
+
+[[repo]]
+url = "https://github.com/ahoym/cra-xrpl-sandbox"
+
+[[repo]]
+url = "https://github.com/Reacttt/303COM_FYP"
+
+[[repo]]
+url = "https://github.com/maltablock/pricefeed"
+
+[[repo]]
+url = "https://github.com/findcoo/BitGoJS"
+
+[[repo]]
+url = "https://github.com/archipelago-org/dashboard"
+
+[[repo]]
+url = "https://github.com/xdv-testing/ripple-hashes"
+
+[[repo]]
+url = "https://github.com/selcukemiravci/CharityDAO"
+
+[[repo]]
+url = "https://github.com/SecuX/secux-js"
+
+[[repo]]
+url = "https://github.com/rossgeesman/cryptodex"
+
+[[repo]]
+url = "https://github.com/futureversecom/trn-examples"
```

Comment:
I've been seeing some repos which I'm not sure are building on top of Ripple. Take this as an example, could you explain the rationale or algorithm you are using to find these repos? Thinking how to filter them a bit. I have one idea: feel free to ping me on telegram (same handle as my gh user)

Reply:
hey @eherrerosj! Thanks for flagging that. These are part of the larger XRP Ledger ecosystem. The futureverse (Root Network) is a type of sidechain to the XRP Ledger and thus I included it. I will ping you on telegram if you would like to discuss it further, thanks!